src/axom/quest/SignedDistance.hpp

```diff
@@ -263,19 +262,51 @@ class SignedDistance
    *  everywhere. Specifically, the sign is ambiguous for all points for which
    *  a normal projection onto the surface does not exist.
    *
-   * \return minDist the signed minimum distance to the surface mesh.
+   * \return the signed minimum distance to the surface mesh.
    */
   double computeDistance(const PointType& queryPnt) const;
 
+  /*!
+   * \brief Computes the distance of the given point to the surface mesh
+   * This overload also returns the computed closest point on the surface to the query point
+   * and the surface normal at that point
+   *
+   * \param [in] queryPnt user-supplied point.
+   * \param [out] closestPnt the closest point on the surface to \a queryPnt
+   * \param [out] surfaceNormal the surface normal of \a closestPt
+   *
+   * \note When the input is not a closed surface mesh, the assumption is that
```

Comment:
Nitpicky here, but I don't see any function parameter that can be a surface.  by "input", I'm assuming you mean the surfaceMesh in the constructor.

Reply:
clarified